Makefile

```diff
@@ -7,4 +7,9 @@ check-password-strength:
 install:
 	go install ./cmd/chain-indexing/
 migrate:
-	./pgmigrate.sh -- -verbose up	
+	./pgmigrate.sh -- -verbose up
+
+.PHONY: docker
+docker:
+	cp .gitignore .dockerignore && echo "**/.git" >> .dockerignore
```

Comment:
We only need to copy the code & config to docker image, ignore the useless files.
Files ignored by git are also ignored by docker, so reuse the `.gitignore` file, or we need to maintain `.dockerignore` file
https://docs.docker.com/engine/reference/builder/#dockerignore-file

Reply:
Understood but not all the file ignored git should be ignored e.g. config/config.toml. So may be we can have a dedicated .dockerignore rather than coping each time

Reply:
@davcrypto I've added `.dockerignore`, we mount config to docker with docker run, so we don't care config files, we can easily modify the configuration without having to rebuild images every time the configuration is changed.
```
docker run \
    -v `pwd`/config:/app/config --read-only \
    -p 28857:28857 \
    --env DB_PASSWORD=your_postgresql_password \
    chain-indexing /app/chain-indexing
```